contrib/aerospike/aerospike-client.v5/option.go

```diff
@@ -0,0 +1,65 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package aerospike
+
+import (
+	"math"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal"
+)
+
+const (
+	serviceName   = "aerospike"
+	operationName = "aerospike.command"
+)
+
+type clientConfig struct {
```

Comment:
Since there is only one type of config, mind calling this `config`?

Reply:
I can, but the same thing exist in the memcache one. There is a single config and it's called `clientConfig`. Should I change?